### PR TITLE
Feature/cross language API revision context

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Cross Language View Implementation
resolves https://github.com/orgs/Azure/projects/197/views/14?pane=issue&itemId=9565805

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Helpers/PageModelHelpers.cs [hunk 1]
```diff
@@ -524,6 +535,15 @@ public static async Task<APIRevisionListItemModel> UploadAPIRevisionAsync(IAPIRe
             }
         }
         
+        /// Deside if the content of the API is the header or content
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Helpers/PageModelHelpers.cs:538]:
```suggestion
        /// Decide if the content of the API is the header or content
```

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs [hunk 2]
```diff
@@ -90,6 +90,23 @@ public async Task<IActionResult> OnGetAsync(string id, string revisionId = null)
                 showDocumentation: ShowDocumentation, showDiffOnly: ShowDiffOnly, diffContextSize: REVIEW_DIFF_CONTEXT_SIZE,
                 diffContextSeperator: DIFF_CONTEXT_SEPERATOR);
 
+            if (!String.IsNullOrEmpty(ReviewContent.ActiveAPIRevision.Files.First().CrossLanguagePackageId))
+            {
+                var correspondingReviewId = await _apiRevisionsManager.GetReviewIdsOfLanguageCorrespondingReviewAsync(ReviewContent.ActiveAPIRevision.Files.First().CrossLanguagePackageId);
+                var correspondingReviews = await _reviewManager.GetReviewsAsync(reviewIds: correspondingReviewId, isClosed: false);
+                foreach (var review in correspondingReviews)
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs:97]: Will those cause perf issue when loafing all related reviews? This looks like caching all related review contents. right?

**Review by praveenkuttappan — APPROVED** [review]
Looks fine except the question about a performance when loading a review.

File: src/dotnet/APIView/APIViewWeb/Helpers/PageModelHelpers.cs [hunk 3]
```diff
@@ -524,6 +535,15 @@ public static async Task<APIRevisionListItemModel> UploadAPIRevisionAsync(IAPIRe
             }
         }
         
+        /// Decide if the content of the API is the header or content
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/APIViewWeb/Helpers/PageModelHelpers.cs:538]:
```suggestion
        /// <summary>
        /// Decide if the content of the API is the header or content
```

File: src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs (hunk 2 above)
Comment (chidozieononiwu) [reply src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs:97]: Potentially. But for regular sized reviews that I have tested it is performing fine.

**Pull request merged**